generators/server/templates/src/main/java/package/config/_DefaultProfileUtil.java

```diff
@@ -20,14 +18,7 @@
      * @param app the spring application
      */
     public static void addDefaultProfile(SpringApplication app) {
-        Map<String, Object> defProperties =  new HashMap<>();
-        /*
-        * The default profile to use when no other profiles are defined
-        * This cannot be set in the <code>application.yml</code> file.
-        * See https://github.com/spring-projects/spring-boot/issues/1219
-        */
-        defProperties.put(SPRING_PROFILE_DEFAULT, Constants.SPRING_PROFILE_DEVELOPMENT);
-        app.setDefaultProperties(defProperties);
+        app.setAdditionalProfiles(Constants.SPRING_PROFILE_DEVELOPMENT, Constants.SPRING_PROFILE_SWAGGER);
```

Comment:
@jdubois If we run by default this would cause a double loading of `dev` profile as it would have been already loaded from the `application.yml` file once. If we run in prod, it will also load `dev` which is very bad. If we decide to do this conditionally at this point we would not have any profile info in the first place to decide if we need to set this or not that is why I went with `app.setDefaultProperties(defProperties);` as spring would disregard it when there is a profile set using any of the spring suggested way